internal/pkg/cache/set.go

```diff
@@ -43,11 +44,15 @@ func (c *Set) Get(key string, dest interface{}) error {
 	} else {
 		r = reflect.ValueOf(result)
 	}
+	fmt.Println(result, dest)
+	fmt.Println(reflect.TypeOf(result), reflect.TypeOf(dest))
+	fmt.Println(r.Type(), reflect.TypeOf(dest))
 	reflect.ValueOf(dest).Elem().Set(r)
```

Comment:
are these output removable

Reply:
resolved in 25e820673bce6039989fea3f5a1a437486a8ad44